README.md

```diff
@@ -115,6 +117,32 @@ luigi -m my_module mytask --rerun
 luigi -m my_module another_task_that_depends_on_mytask --MyTask-rerun
 ```
 
+### Clear the output of failed tasks
+
+When a task fails unexpectedly, it may leave an incomplete or corrupted output 
+that leads to wrong results in the downstream. With the RemoveCorruptedOutputMixin,
+Luigi automatically removes the output of the tasks that failed. This is the default behaviour 
+in other workflow management systems such as [Snakemake][snakemake_url].
+
+```python
+from luigi_tools.task import RemoveCorruptedOutputMixin
+
+class ResilientTask(RemoveCorruptedOutputMixin, luigi.Task):
+    """Resilient to the bugs caused by incomplete and corrupted outputs."""
+
+    RemoveCorruptedOutputMixin.clean_failed = luigi.BoolParameter(
+        significant=False,
+        default=True,
+        description="Trigger to remove the outputs of the failed tasks.",
+        parsing=luigi.BoolParameter.EXPLICIT_PARSING,
+    )
```

Comment:
The default behaviour was True in my first PR. Though you convinced me to make it optional. I don't remember what the motivation was.

Reply:
> Maybe it would be simpler to remove this and show an example of CLI command that triggers this behavior? 

Then there is also no need to have the ResilientTask class, right? 

TaskA can directly use the mixin like this.

`class TaskA(RemoveCorruptedOutputMixin, luigi.Task):`

Reply:
Yes exactly.

Reply:
> The default behaviour was True in my first PR. Though you convinced me to make it optional. I don't remember what the motivation was.

Ahah ok, I don't remember what was the argument. I am going to find out.

Reply:
Ah it's an old PR that is no more accessible on Gerrit :'(
I think the point was that the default behavior should stick to the default Luigi behavior to not confuse the user. And as you showed in your example, it is still easy to reverse the behavior if someone wants to.
So I think that having a simpler example with the luigi command is enough. Do you agree?

Reply:
Ok sounds good.

Reply:
Ah yes. And such default also makes things easier for a user to send the failed state to the support.